src/pages/components/NewComment.jsx

```diff
@@ -0,0 +1,70 @@
+import { useState } from 'react';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import InputGroup from 'react-bootstrap/InputGroup';
+import axios from 'axios';
+
+const NewComment = ({ article_id, baseURL, setCommentList }) => {
+  const [username, setUsername] = useState('');
+  const [commentBody, setCommentBody] = useState('');
+  const [isSubmitting, setSubmitting] = useState(false);
+  const [isError, setError] = useState(false);
+
+  const handleNewComment = async (e) => {
+    e.preventDefault();
+    setSubmitting(true);
+    setError(false);
+    try {
+      const response = await axios.post(
+        `${baseURL}articles/${article_id}/comments`,
+        {
+          author: username || 'Anon',
+          body: commentBody,
+        }
+      );
```

Comment:
This part can be bundled up into a function instead to make this a little more readable